From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Greg Kroah-Hartman <gregkh@google.com>
Date: Sat, 13 Mar 2021 17:30:11 +0100
Subject: ANDROID: GKI: remove CONFIG_CMDLINE_EXTEND from arm64 gki_defconfig

In commit cae118b6acc3 ("arm64: Drop support for CMDLINE_EXTEND"), this
configuration option was removed, so also remove it from the arm64
gki_defconfig so that the build continues to work properly.

Fixes: cae118b6acc3 ("arm64: Drop support for CMDLINE_EXTEND")
Cc: Will Deacon <willdeacon@google.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@google.com>
Change-Id: I1d6570e840134efc62d7c9ab4a25b63725999e0d
---
 arch/arm64/configs/gki_defconfig | 1 -
 1 file changed, 1 deletion(-)

diff --git a/arch/arm64/configs/gki_defconfig b/arch/arm64/configs/gki_defconfig
--- a/arch/arm64/configs/gki_defconfig
+++ b/arch/arm64/configs/gki_defconfig
@@ -59,7 +59,6 @@ CONFIG_SETEND_EMULATION=y
 CONFIG_RANDOMIZE_BASE=y
 # CONFIG_RANDOMIZE_MODULE_REGION_FULL is not set
 CONFIG_CMDLINE="stack_depot_disable=on kasan.stacktrace=off"
-CONFIG_CMDLINE_EXTEND=y
 # CONFIG_DMI is not set
 CONFIG_PM_WAKELOCKS=y
 CONFIG_PM_WAKELOCKS_LIMIT=0
